assets/charts/control-plane/calico/templates/felixconfig.yaml

```diff
@@ -0,0 +1,11 @@
+# General
+# =======
+---
+# Disable default failsafe inbound rules
+apiVersion: crd.projectcalico.org/v1
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  failsafeInboundHostPorts: []
```

Comment:
This option should be enabled only if we have some GNPs in place, otherwise cluster will loose all the connectivity I think. It should be optional, hidden by bootkube module variable, which Packet will set to true.

Reply:
@invidian Do you think this is in scope of this PR? I have no problem adding it as a different commit, just wanted to know.

Reply:
Yes, that should be done as part of moving the configuration to bootkube module.